src/content/en/showcase/2017/with.in.md

```diff
@@ -0,0 +1,331 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2017-02-02T18:00:00.000Z #}
+{# wf_updated_on: 2017-02-02T18:00:00.000Z #}
+{# wf_featured_image: /web/showcase/2017/images/with.in/featured.png #}
+{# wf_featured_snippet: Within is a platform for storytelling in Virtual Reality and is available everywhere VR is. This includes the web. Leveraging WebVR, viewers are able to go to the website, click a link, and immediately watch Within’s films in immersive VR - including high end head-mounted displays. During the development process the team discovered rendering text is difficult in this new environment, and they created an example using shaders to make it a smoother process. #}
+{# wf_tags: webvr,casestudy #}
+
+
+# Rendering Text in WebVR {: .page-title }
+
+**Project:** [https://vr.with.in/](https://vr.with.in/)
+
+**Demo:** [https://vr.with.in/archive/text-sdf-bitmap/](https://vr.with.in/archive/text-sdf-bitmap/)
+
+**Working Files:** [https://vr.with.in/archive/text-sdf-bitmap.zip](https://vr.with.in/archive/text-sdf-bitmap.zip)
+
+<img src="/web/showcase/2017/images/with.in/0.png">
+
+## Introduction
+
+Within ([http://with.in/](http://with.in/)) is a platform for storytelling in Virtual Reality. So when the team heard about WebVR in 2015 we were immediately interested in its potential. Today that interest manifests into a unique subdomain of our Web Platform, [https://vr.with.in/](https://vr.with.in/. Anyone with a VR Enabled Browser can go to the site click a button and throw on a headset to be immersed in our portfolio of VR films.
+
+*Today that includes but is not limited to Chrome on Daydream View. For information on your device and head-mounted display check out [https://webvr.info/](https://webvr.info/).*
+
+Like other Virtual Reality specific rendering environments, the web predominantly relies on a three-dimensional representation of a scene. This scene has a camera, your perspective, and any number of objects. To help manage this scene, camera, and objects we use a library called [Three.js](http://threejs.org/) which leverages the `<canvas>` element to throw rendering onto your computer's GPU. There are many useful Three.js add-ons to make your scene viewable in WebVR. The main two are [THREE.VREffect](https://github.com/mrdoob/three.js/blob/dev/examples/js/effects/VREffect.js) for creating a viewport for each eye and [THREE.VRControls](https://github.com/mrdoob/three.js/blob/dev/examples/js/controls/VRControls.js) for translating the perspective (e.g the rotation and position of the head-mounted display) convincingly into your scene. There are many examples of how to implement this. Check out [https://threejs.org/examples/?q=webvr](https://threejs.org/examples/?q=webvr) for ways to get started.
```

Comment:
Last link, would be better to have text for the link. Three.js WebVR examples for instance?